02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -1,28 +1,34 @@
 # State of PureScript Documentation: 2019
 
-## Why Read This Document?
+This document is not meant to convince you to use PureScript, we assume that readers are already interested in it.
```

Comment:
I'm restating what I already stated to make sure my prior comment doesn't get overlooked.

I think this should be rendered as "we assume that readers are already interested in or use it.' The 'already interested in' refers to new learners. The 'use' refer to users and core contributors.

Reply:
I was thinking "interested people" includes users and contributors, actually. If they weren't interested in it, they wouldn't be using or contributing to it, right? But yeah, it's true that people could be coerced into using PureScript, such as for a school project or for work, so yeah your phrasing is probably better.